### ISSUE TITLE
[LSPDFR/RPH] MenyooSP cause Game Crash with LSPDFR/RPH 

### ISSUE THREAD

**Issue by unitedOrange66** [MAFINS/MenyooSP#487]
Along with LSPDFR and RPH installed, whenever the player is near Mission Row Police Station, the game would freeze then crash to the desktop.

**Comment by itsjustcurtis** [comment]
Have you confirmed the issue goes away when menyoo is uninstalled whilst LSPDFR and RPH remain active?

**Comment by unitedOrange66** [comment]
> Have you confirmed the issue goes away when menyoo is uninstalled whilst LSPDFR and RPH remain active?

Yes. I can confirm that. I've been playing LSPDFR via RPH for months without Menyoo, it works just fine.

**Comment by unitedOrange66** [comment]
Add to that ^ 

When Menyoo doesn't cause any crashes with LSPDFR and RPH back to 2019 - 2020, it does cause a weird issue from time to time, where empty police cars would spawn in the middle of the road randomly; and multiplayer model AI cops would be "bugged" at the mission row police station.(not spanning with proper uniforms, standing around all over the place without doing anything)

This issue can be confirmed by other LSPDFR players who have used Menyoo. However it didn't get too much attention since it happens randomly and most people tend to ignored it
